packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -60,8 +60,8 @@ Requires: %{?scl_prefix}rubygem(pulp_rpm_client) >= 3.6.2
 Requires: %{?scl_prefix}rubygem(pulp_rpm_client) < 3.9.0
 Requires: %{?scl_prefix}rubygem(pulp_deb_client) >= 2.6.0
 Requires: %{?scl_prefix}rubygem(pulp_deb_client) < 2.9.0
-Requires: %{?scl_prefix}rubygem(pulp_2to3_migration_client) >= 0.3.0
-Requires: %{?scl_prefix}rubygem(pulp_2to3_migration_client) < 0.7.0
+Requires: %{?scl_prefix}rubygem(pulp_2to3_migration_client) >= 0.7.0
+Requires: %{?scl_prefix}rubygem(pulp_2to3_migration_client) < 0.8.0
```

Comment:
Also the build requires needs this update

Reply:
updated!